samples/efcpt-schema.json

```diff
@@ -457,13 +457,22 @@
                     "examples": [
                         false
                     ]
+                },
+                "use-schema-namespaces-preview": {
+                    "type": "boolean",
+                    "default": false,
+                    "title": "Use schema namespaces (preview)",
+                    "examples": [
+                        false
+                    ]
                 }
             },
             "examples": [{
                 "output-path": "Models",
                 "output-dbcontext-path": "Context",
                 "split-dbcontext-preview": false,
-                "use-schema-folders-preview": false
+                "use-schema-folders-preview": false,
+                "use-schema-namespaces-preview": false,
```

Comment:
I think this is invalid json? (Comma at the end)

Reply:
I'm sorry